src/ngx_postgres_rewrite.c

```diff
@@ -322,28 +322,28 @@ ngx_postgres_rewrite(ngx_http_request_t *r,
                         ngx_str_t html_variable = ngx_string("html");
                         ngx_uint_t html_variable_hash = ngx_hash_key(html_variable.data, html_variable.len);
                         ngx_http_variable_value_t *raw_html = ngx_http_get_variable( r, &html_variable, html_variable_hash  );
-                        
+
                         raw_html->len = rewrite[i].location.len;
                         raw_html->data = rewrite[i].location.data;
 
                         // bad request 400 on errors
-                        // if i return 400 here, pg result is lost :( YF: FIXME 
+                        // if i return 400 here, pg result is lost :( YF: FIXME
                         if (pgrcf->key % 2 == 1 && pgrcf->handler == &ngx_postgres_rewrite_valid) {
                           return 200;
                         } else {
                           return 200;
                         }
                     // redirect to outside url
                     } else {
-                        // errors/no_errors rewriters already provide interpolated url, 
+                        // errors/no_errors rewriters already provide interpolated url,
                         // but others need to do it here
                         if (url == NULL) {
                           char *variables[10];
                           char *columned[10];
                           char *values[10];
                           ngx_postgres_ctx_t  *pgctx = ngx_http_get_module_ctx(r, ngx_postgres_module);
                           int vars = ngx_postgres_find_variables(variables, (char *) rewrite[i].location.data, rewrite[i].location.len);
-                          char *error = ngx_postgres_find_values(values, variables, vars, columned, pgctx, 1);
+                          ngx_postgres_find_values(values, variables, vars, columned, pgctx, 1);
```

Comment:
The *error variable is not used anywhere else and therefore generates a compiler warning. However, I wasn't sure if ngx_postgres_find_values() has any intended side effects, so I decided to keep it. Perhaps the whole call to ngx_postgres_find_values can be removed.